db/migrate/20221115082403_add_ldap_tls_options.rb

```diff
@@ -9,9 +9,7 @@ def change
       dir.up do
         # Current LDAP library default is to not verify the certificate
         LdapAuthSource.reset_column_information
-        ldap_settings = (Setting.ldap_tls_options || {}).with_indifferent_access
-        verify_peer = ldap_settings[:verify_mode] == OpenSSL::SSL::VERIFY_PEER
-        LdapAuthSource.update_all(verify_peer:)
+        LdapAuthSource.update_all(verify_peer: false)
```

Comment:
Why is this change part of the PR

Reply:
because ldap_tls_options is not in use anymore after 12.4., and I thought it was safe to remove. But of course if somehow skips the 12.4 release cycle, they might get different results if the had the setting active.

We can keep it in until 13.=ß